pkg/api/v1/clients/vault/resource_client.go

```diff
@@ -70,13 +70,15 @@ func (rc *ResourceClient) toVaultSecret(resource resources.Resource) (map[string
 type ResourceClient struct {
 	vault        *api.Client
 	root         string
+	pathPrefix   string
 	resourceType resources.VersionedResource
 }
 
-func NewResourceClient(client *api.Client, rootKey string, resourceType resources.VersionedResource) *ResourceClient {
+func NewResourceClient(client *api.Client, rootKey string, pathPrefix string, resourceType resources.VersionedResource) *ResourceClient {
```

Comment:
True, thanks! I'll update the changelog for a new major version. I'll also probably move `pathPrefix` before `rootKey`, since that's the way the `resourceDirectory` the client works with is created.